inference-engine/ie_bridges/python/src/openvino/inference_engine/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ endfunction()
 
 python_disable_deprecated_warnings()
 
-target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}")
+target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}" ${IE_MAIN_SOURCE_DIR}/src/inference_engine)
```

Comment:
Personally, dislike the fact to use something from internal API

Reply:
This solution is temporary until we have no Python API for IR Frontend. Also we need IE and NG python API to be untied.

Reply:
Removed.